htdocs/commande/class/commande.class.php

```diff
@@ -285,6 +285,8 @@ class Commande extends CommonOrder
 	 */
 	public $online_payment_url;
 
+	public $stocks = array();
```

Comment:
It seems the property ->stocks is used only in method public function stock_array() of the object Commande.
And a scan of code show that this method is never called. So i suggest to not add this property and instead to completely comment the dead method...